src/phoenix/trace/dsl/filter.py

```diff
@@ -40,15 +43,18 @@ class AliasedAnnotationRelation:
 
     index: int
     name: str
+    annotation_type: str
```

Comment:
where is this variable being used?

Reply:
we're only searching on SpanAnnotations right now and leaving the filter identical to `evals` for the time being, I left this as space for filtering by by AnnotatorKind since explicitly making that distinction is something we're considering

Reply:
Unless you disagree, I would recommend that we add things only when they're being used, lest we start a precedent of preemptively adding things before they are actually used, however obviously useful the things may be.